scheduling/multi_level_feedback_queue.py

```diff
@@ -0,0 +1,319 @@
+from collections import deque
+
+
+class Process:
+    """
+    Process should have these fields below
+    """
```

Comment:
it's ok to remove

Reply:
Thanks for your suggestions!😃😃
I'm sorry for bothering you cause my English skill is not good.